bin/all_sky_search/pycbc_fit_sngls_split_binned

```diff
@@ -236,19 +322,24 @@ if args.gating_veto_windows:
         raise ValueError("Gating veto window values must be negative before "
                          "gates and positive after gates.")
     if not (gveto_before == 0 and gveto_after == 0):
-        time = trigf[args.ifo + '/end_time'][:]
         autogate_times = np.unique(trigf[args.ifo + '/gating/auto/time'][:])
         if args.ifo + '/gating/file' in trigf:
             detgate_times = trigf[args.ifo + '/gating/file/time'][:]
         else:
             detgate_times = []
         gate_times = np.concatenate((autogate_times, detgate_times))
-        gveto_remove = events.veto.indices_within_times(time, gate_times + gveto_before,
-                                                        gate_times + gveto_after)
-        stat[gveto_remove] = np.zeros_like(gveto_remove)
-        time[gveto_remove] = np.zeros_like(gveto_remove)
-        logging.info('{} out of {} trigs removed after vetoing triggers near gates'.format(
-                          gveto_remove.size, stat.size))
+        gveto_remove = events.veto.indices_within_times(
+            time,
+            gate_times + gveto_before,
+            gate_times + gveto_after
+        )
+        stat[gveto_remove] = 0.
```

Comment:
Is this just a simplification of unnecessarily complicated code?

Reply:
The only functional change is to do `stat[gveto_remove] = 0.` instead of `stat[gveto_remove] = np.zeros_like(gveto_remove)`. As `stat[gveto_remove]` is a numpy array *view*, I think it duplicates a bunch of memory to use `np.zeros_like` here. The old code creates a new array, and then slices it up to assign specific values into the `stat` array. The new code just sets a bunch of values to 0. Quicker and more memory efficient .... But this part of the code is not important to the overall cost, I just didn't like it!

Reply:
Oh sure, that is what I expected.